core/src/main/java/nl/nn/adapterframework/compression/ZipWriterPipe.java

```diff
@@ -69,4 +136,14 @@ public void setZipWriterHandle(String string) {
 	public void setCompleteFileHeader(boolean b) {
 		includeFileHeaders = b;
 	}
+
+	/**
+	 * When action is OPEN: If input is a string, it's assumed it's the location where to save the Zip Archive.
+	 * When action is WRITE: Input will be 'piped' to the output, and the message will be preserved.
+	 * Avoid using this if possible.
+	 */
+	@Deprecated
+	public void setBackwardsCompatibility(boolean backwardsCompatibility) {
+		this.backwardsCompatibility = backwardsCompatibility;
```

Comment:
Ik vind dit een beetje knullig maar de iaf-test scenario's (die overigens niet inhoudelijk iets testen) verwachten dat de input de output is. Die kan aangepast worden door de tests aan te passen maar daar heb ik niets aan gedaan om 'aantoonbaar' te maken dat het wel compatible is...

Reply:
Deze vlag moet ook worden toegevoegd aan de release notes, en ook wanneer je de vlag wel moet zetten en wanneer dat niet nodig is.

Reply:
- The ZipWriterPipe with action=WRITE does no longer have its input as its response, but rather a null message. If necessary, the previous behaviour can be obtained by setting preserveInput=true.
- The ZipWriterSender with action=WRITE (the default) and no content parameter does no longer have its input as its response, but rather a null message.

staat al in de release notes, maar ik zal ook het attribuut noemen!